steps/data_scraping_steps/scrape_nhs_data/scrape_nhs_data_step.py

```diff
@@ -151,6 +171,7 @@ def scrape_nhs_data() -> Annotated[pd.DataFrame, "output_nhs_data"]:
         attributes={"class": "nhsuk-main-wrapper"},
     )
     nhs_scraper.scrape_recursively()
+    nhs_scraper.discard_non_content()
 
     nhs_scraper.df.to_csv(os.path.join(DATA_DIR, "nhs_data_raw.csv"))
```

Comment:
Nit: Can we add `index=False` inside `to_csv` for both Mind and NHS? After running the `data_scraping` pipeline, we also get index column saved into csv.

```python
print(df.columns)

Index(['Unnamed: 0', 'uuid', 'html_scraped', 'text_scraped', 'timestamp',
       'url'],
      dtype='object')

```

Reply:
An enhancement as part of separate ticket would be removing `uuid` column as well as we don't use it in downstream tasks.

Reply:
`index=false` would be great

Reply:
It was added for chroma store before chunking was in place. Unique ids per text are required for chroma while inserting text into chroma store.